tamars_cookies.md

```diff
@@ -0,0 +1,42 @@
+# Ingredients
+
+
+
+```
+45 grams all-purpose flour
+30 grams Dutch-processed cocoa powder
+¾ teaspoon baking powder
+½ teaspoon kosher salt
```

Comment:
Nit: Can you replace the fraction symbols with decimals? 

i.e.) ½ -> 0.5

Reply:
No.